code/modules/projectiles/guns/projectile.dm

```diff
@@ -371,5 +371,5 @@
 		if(CAL_PISTOL)
 			gun_tags |= GUN_CALIBRE_35
 		//Others to be implemented when needed
-	if(max_shells)
+	if(max_shells > 1) // so the 
```

Comment:
so the?

Reply:
forgot this existed, was supposed to make it impossible to attach the overshooter to an AMR